### PR TITLE
Small Cellpose improvements

### PR DESCRIPTION
**Pull request by hugokallander**
- Added ensure_3_channels_batch
- Added validation performance logging during training
- Made test, instead of prod, deployment default. **TODO**: replace ri-scale workspace with bioimage-io after given access

### CODE REVIEW

File: bioengine_apps/cellpose_finetuning/training_example_runs/training_metrics_d679cca1.tsv
Comment (oeway) [inline bioengine_apps/cellpose_finetuning/training_example_runs/training_metrics_d679cca1.tsv]: this should not be commited I guess.

File: bioengine_apps/cellpose_finetuning/scripts/redeploy_cellpose.py [hunk 1]
```diff
@@ -45,4 +47,26 @@ async def redeploy():
...
 if __name__ == "__main__":
-    asyncio.run(redeploy())
+    # add argparse argument artifact Id and application id
+    parser = argparse.ArgumentParser(
+        description="Redeploy the cellpose finetuning application with the latest code.",
+    )
+    parser.add_argument(
+        "--artifact-id",
+        type=str,
+        default="ri-scale/cellpose-finetuning-test",
```
Comment (oeway) [inline bioengine_apps/cellpose_finetuning/scripts/redeploy_cellpose.py]: the default should not be ri-scale workspace, it should be `bioimage-io/cellpose-finetuning`

Comment (hugokallander) [reply bioengine_apps/cellpose_finetuning/scripts/redeploy_cellpose.py]: Ah sorry, I misunderstood. The default should never have to be ri-scale. I'll fix this right away

File: bioengine_apps/cellpose_finetuning/main.py [hunk 2]
```diff
@@ -849,13 +929,18 @@ def train_seg_with_callbacks(
                 batch_idx = k // batch_size
                 total_batches = (nimg_per_epoch + batch_size - 1) // batch_size
                 batch_loss_per_sample = loss.item()  # Loss per sample for this batch
-                batch_callback(iepoch + 1, batch_idx, total_batches, batch_loss_per_sample, elapsed)
+                batch_callback(
```
Comment (oeway) [inline bioengine_apps/cellpose_finetuning/main.py]: I am not sure, but would it make sense to let batch_callback also carry another argument for the test metrics? and can be None for training but with value for validation? does that make sense? I am not sure though.
